### ISSUE TITLE
Breaks plasma 6 global shortcut service.

### ISSUE THREAD

**Issue by zany130** [dnut/clipboard-sync#36]
somehow the user systemd service this creates breaks plasma 6 global shortcut service. I vertified this by removing `/home/zany130/.config/systemd/user/graphical-session.target.wants/clipboard-sync.service`.

before the removal  adding a shortcut to a script in plasma settings produces this error in the terminal

```
kcmshell6 kcm_keys
qt.qml.typeresolution.cycle: Cyclic dependency detected between "qrc:/qt/qml/org/kde/desktop/private/TextFieldContextMenu.qml" and "qrc:/qt/qml/org/kde/desktop/MenuItem.qml"
qrc:/kcm/kcm_keys/main.qml:466:5: QML OverlaySheet: Binding loop detected for property "implicitHeight"
qrc:/qt/qml/org/kde/desktop/private/TextFieldContextMenu.qml:235:5: QML MenuItem: Binding loop detected for property "implicitWidth"
org.kde.kcm_keys: "Error while calling objectPath of added applicationnotify-send.desktop"
org.kde.kcm_keys: "org.kde.kglobalaccel.NoSuchComponent" "The component 'notify-send.desktop' doesn't exist."
```
removing the file allows me to successfully add the same shortcut.

I tried reinstalling the clipboard-sync package both from the aur and by https://github.com/dnut/clipboard-sync#advanced-installation 

and it still breaks.

here is the KDE issue where I was troubleshooting this
https://discuss.kde.org/t/unable-to-add-shortcuts-plasma-6-regression/13383

**Comment by dnut** [comment]
Thanks for the report.

All you did was delete that file and it resolved the issue? Did you also stop the service or reboot?

That file is not actually a part of clipboard-sync. It's created by systemd when you enable the clipboard-sync service. When the service is enabled, that means it will start automatically when you login. But you can actually run the service without enabling it.

I'm curious if the problem is actually specifically due to that file, or if it's simply caused by clipboard-sync running.

With the service disabled, can you try running the `clipboard-sync` command and see if you can reproduce that error while clipboard-sync is running?

**Comment by zany130** [comment]
Yes you are correct. its not the service file that causes the issue its the actual process. running `systemctl --user stop clipboard-sync.service` also allows me to add shortcuts to kde. trying to run `clipboard-sync --log-level trace` to see if I can find anything usefull
